docs/modules/deploy/pages/versioning-compatibility.adoc

```diff
@@ -135,16 +135,13 @@ Hazelcast Platform has been tested against the following JVMs. It may run on oth
 |===
 |JDK | Platform {full-version}
 
-|Amazon Corretto 17
-|✓
-
-|Amazon Corretto 11
+|AdoptOpenJDK 11
```

Comment:
Where did this come from? We don't test with `AdoptOpenJDK 11`

Reply:
Copy paste error (from an old google docs)